drizzlepac/haputils/pandas_utils.py

```diff
@@ -35,6 +35,45 @@
 DETECTOR_LEGEND = {'UVIS': 'magenta', 'IR': 'red', 'WFC': 'blue',
                    'SBC': 'yellow', 'HRC': 'black'}
 
+def get_pandas_data(pandas_filename, data_columns, log_level=logutil.logging.NOTSET):
+    """Load the harvested data, stored in a CSV file, into local arrays.
```

Comment:
Change CSV -> HDF5.
Please change CSV to HDF5 all locations as appropriate.

Reply:
Addressed in 09f65ce9